test/containers/uploadButton.spec.jsx

```diff
@@ -1,84 +0,0 @@
-'use strict'
```

Comment:
Well, `UploadButton` isn't a container anymore, so these tests are obsolete. Pushing further the tests on `UploadButton` would require browser testing, because what's important in this component is its DOM behavior.